mining_tool_instruction/windows_defender/en.html

```diff
@@ -3,4 +3,6 @@
 3. In the Exclusions section, click Add or remove exclusions
 4. Click Add an exclusion, then select folder
 {virus_threat_protection_screenshot}
-5. Navigate to this path, or paste it to the address bar
\ No newline at end of file
+5. Navigate to this path, or paste it to the address bar
+- Change `user_folder` by your folder name
+- C:\Users\user_folder\AppData\Roaming\miner.crypto.tm\miners
```

Comment:
Ugh... this should be specific based on the User? We should generate this from the code to provide exact location.

Reply:
Ahhh so we want it to actually detect the path. Right, we thought only generic